app/src/main/java/org/fossasia/openevent/general/auth/ProfileFragment.kt

```diff
@@ -51,11 +52,19 @@ class ProfileFragment : Fragment() {
 
     override fun onStart() {
         super.onStart()
-        if (!profileViewModel.isLoggedIn()) {
-            redirectToLogin()
+        if (profileViewModel.isLoggedIn()) {
+            handleLayoutVisibility(true)
+        } else {
+            handleLayoutVisibility(false)
```

Comment:
> if (profileViewModel.isLoggedIn()) {
            handleLayoutVisibility(true)
        } else {
            handleLayoutVisibility(false)
>
This can be written as a single line like `handleLayoutVisibility(profileViewModel.isLoggedIn())`. I think it is cleaner

Reply:
Oh, thanks for pointing. :+1: